### PR TITLE
[SW-1340] fix: be able to open a tip deeplink on mobile

### PR DESCRIPTION
**Pull request by CedrikNikita**

### CODE REVIEW

File: src/composables/deepLinkApi.ts [hunk 1]
```diff
@@ -26,8 +28,11 @@ export function useDeepLinkApi({ router }: UseDeepLinkApiOptions) {
   /**
    * Function needed to support legacy tipping from superhero.com
    */
-  function checkIfOpenTransferSendModal() {
-    if (checkIfSuperheroCallbackUrl(route.query)) {
+  async function checkIfOpenTransferSendModal(currentRoute: Route) {
+    if (
+      currentRoute.path.slice(1) === ROUTE_ACCOUNT
```
Comment (pstachel) [inline src/composables/deepLinkApi.ts:33]: Should we open the transfer modal only on the account page?

Comment (pstachel) [inline src/composables/deepLinkApi.ts:33]: Wouldn't `currentRoute.name` resolve the need of using slice?

**Review by pstachel — APPROVED** [review]

File: src/composables/deepLinkApi.ts (hunk 1 above)
Comment (CedrikNikita) [reply src/composables/deepLinkApi.ts:33]: `name` is `undefined` in this case.

Comment (CedrikNikita) [reply src/composables/deepLinkApi.ts:33]: This check is used only on the `/account` page. And the legacy code we are trying to support is aiming only for this page.

**Comment by Liubov-crypto** [comment]
LGTM

**Review by martinkaintas — APPROVED** [review]

**Review by peronczyk — APPROVED** [review]

**Pull request merged**